lib/config/configuration_test.go

```diff
@@ -2156,3 +2172,135 @@ func TestTLSCert(t *testing.T) {
 		})
 	}
 }
+
+func Test_applyKeyStoreConfig(t *testing.T) {
```

Comment:
```suggestion
func TestApplyKeyStoreConfig(t *testing.T) {
```

Reply:
👍  Wasn't too sure what the convention was here. I'm used to having test names that match the name of the unit under test.